### ISSUE TITLE
Skip Navigation Links 

### ISSUE THREAD

**Issue by carinaluise** [substrate-developer-hub/substrate-docs#1301]
### Skip Navigation Links 

On most pages, keyboard and screen reader users must navigate a long list of navigation links and other elements before arriving at the main content. This can be particularly difficult for users with some forms of motor disabilities. Skip navigation links can help improve accessibility.

A skip navigation link is helpful for users who don't want to work their way through all the links on the page before getting to the main content. This link does not have to be visible on the page, but it should be listed first in the tab order.

As substrate docs is navigation heavy it would be advisable to add in a skip navigation link.
Example of implementing a skip navigation that is hidden until user navigates with a keyboard: https://webaim.org/techniques/skipnav/

**Comment by jonnysmillie** [comment]
Hi Carina,

With this one I think it is appropriate that the main navigation comes before the documentation navigation in the hierarchy. If it didn't where could it be placed?

The main issue here in my opinion is that the main (shared) navigation exists at all. The real solution to this issue would be to remove it all together, any other change would just be a workaround to accommodate for the keeping of this navigation that is not entirely relevant to the documentation site.

I hope that makes sense. Let me know if there is any further information I can provide.
